src/test/java/io/jenkins/plugins/ml/IPythonKernelInterpreterTest.java

```diff
@@ -0,0 +1,30 @@
+package io.jenkins.plugins.ml;
```

Comment:
missing license header

Reply:
rat plugin reports me to add licences to all files in the repo.

Reply:
Oh interesting. Either it is its configuration, or maybe our header ia missing some text or malformatted I think